Extra imports

ajitsinghkaler opened the pull request:
There are some files which contain extra imports which were not used anywhere in the files.  Removed those extra imports and created a different commit for each file so that if any problem reverting these commit s will be easy. If needed I can squash all these in a commit.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Extra imports

Issue Number: N/A


## What is the new behavior?
Removed extra imports

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

ajitsinghkaler commented:
@JoostK @kara can you please review this

JoostK reviewed (approved):
Thanks for cleaning this up! In this case it's safe for all these changes to go into a single commit, so please squash.

ajitsinghkaler committed:
refactor(compiler): remove extra imports

There are some extra imports in the compiler package. These imports are not used anywhere in the file. So, removed those extra imports

ajitsinghkaler commented:
@JoostK squashed the commits can you please add the merge label

JoostK commented:
@ajitsinghkaler it looks like you'll need to rebase on top of latest master. If you need help with that please let me know.

ajitsinghkaler commented:
@JoostK rebased it

JoostK commented:
merge-assistance: please run presubmit.
